tests/Mailer/MailerSettingsTest.php

```diff
@@ -29,3 +29,17 @@
 	$this->assertSame(10, \has_filter(SettingsMailer::FILTER_SETTINGS_NAME, 'EightshiftForms\Mailer\SettingsMailer->getSettingsData()'), 'The callback getSettingsData should be hooked to custom filter hook with priority 10.');
 	$this->assertSame(10, \has_filter(SettingsMailer::FILTER_SETTINGS_IS_VALID_NAME, 'EightshiftForms\Mailer\SettingsMailer->isSettingsValid()'), 'The callback isSettingsValid should be hooked to custom filter hook with priority 10.');
 });
+
+test('isSettingsValid method returns correct values', function () {
+	$this->assertSame(true, $this->mailerSettings->isSettingsValid('1234'));
+});
+
+test('getSettingsSidebar method returns correct values', function () {
+	expect($this->mailerSettings->getSettingsSidebar())->toBeArray();
+	expect($this->mailerSettings->getSettingsSidebar())->toHaveKeys(['label', 'value', 'icon']);
```

Comment:
Expectations are chainable

```suggestion
	expect($this->mailerSettings->getSettingsSidebar())
		->toBeArray()
		->toHaveKeys(['label', 'value', 'icon']);
```

😉